sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1950,7 +1966,11 @@ SUBROUTINE MDL2P(iostatusD3D)
 !$omp  parallel do private(i,j)
               DO J=JSTA,JEND
                 DO I=1,IM
+                  IF(FSL(I,J)<spval)THEN
```

Comment:
@JesseMeng-NOAA should change to
IF(EGRID1(I,J)<spval)THEN

Reply:
will do

Reply:
will do